tests/testthat/test-old_new.R

```diff
@@ -0,0 +1,37 @@
+context('Old new tests')
+# Verify that hector output for the canonical rcps matches the output from the
+# master branch. This "bit for bit" comparison of the output is intended to help
+# guarantee that Hector's outputs don't change during "minor" and "patch" development.
+# When "major" development is expected to impact Hector output, the comparison data will
+# need to be updated by running data-raw/generate-comp-data.R
+
+test_that("Hector output passes old new test", {
+
+    # In order to pass the old new test the mean aboslute difference between
+    # the old and new  hector output must be less than the defined threshold.
+    error_threshold <- 1e-10
+
+    # Read in the comparison data and extract the information to save.
+    comp_data <- read.csv(list.files('compdata', '.csv',  full.names = TRUE), stringsAsFactors = FALSE)
```

Comment:
You should have a test for the number of files in  your comparison data since you are running `list.files` over something that could be case sensitive.